esp-wifi/src/common_adapter/mod.rs

```diff
@@ -339,3 +348,45 @@ pub(crate) unsafe fn phy_disable_clock() {
         trace!("phy_disable_clock done!");
     }
 }
+
+pub(crate) fn phy_calibrate() {
+    let mut cal_data: [u8; core::mem::size_of::<esp_phy_calibration_data_t>()] =
+        [0u8; core::mem::size_of::<esp_phy_calibration_data_t>()];
+
+    let phy_version = unsafe { get_phy_version_str() };
+    trace!("phy_version {}", unsafe { str_from_c(phy_version) });
+
+    let init_data = &phy_init_data::PHY_INIT_DATA_DEFAULT;
+
+    unsafe {
+        chip_specific::bbpll_en_usb();
```

Comment:
Why is USB required for calibration?

Reply:
we don't need it for calibration but if we start calibration w/o users will loose USB - it's following esp-idf's code (it's done there right before the calibration, too). _Maybe_ we can move it to init but I'm usually a bit scared to do things differently (even if it seems to work it might change less obvious things)

Reply:
If we're doing what idf is doing, it's probably fine to leave it here. Thanks!